pkg/controller/buildrun/buildrun_controller.go

```diff
@@ -195,6 +195,9 @@ func (r *ReconcileBuildRun) Reconcile(request reconcile.Request) (reconcile.Resu
 		updateErr := r.updateBuildRunErrorStatus(ctx, buildRun, err.Error())
 		return reconcile.Result{}, handleError("Failed to fetch the Build instance", err, updateErr)
 	}
+	if build.Status.Registered != "True" {
```

Comment:
Instead of `"True"`, it should be `corev1.ConditionTrue`

Reply:
Thanks, fixed